app/schemas.py

```diff
@@ -16,6 +23,22 @@ class UserBase(BaseModel):
     token: str
 
 
+class ProductCreate(BaseModel):
+    model_config = ConfigDict(from_attributes=True, arbitrary_types_allowed=True)
+
+    code: str
+
+
+class ProductBase(BaseModel):
+    id: int
+    off_source: ProductOFFSource | None
```

Comment:
why do you prefix with off? I don't think that the fact that off is the source is relevant, we can probably skip it :)

Reply:
if we were to have other sources to get product info ?

Reply:
Ok I'll remove the `off_` prefix :ok_hand: